validator/project_validator.go

```diff
@@ -982,6 +982,12 @@ func validateCommands(section string, project *model.Project,
 				Message: fmt.Sprintf("cannot specify both command '%s' and function '%s'", cmd.Command, cmd.Function),
 			})
 		}
+		if cmd.Command == "shell.exec" && cmd.Params["script"] == nil {
```

Comment:
Can you make this a string in globals.go alongside GenerateTasksCommandName to prevent hardcoding